electron/launcher/main-process/config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "DJANGO_SERVER": {
+    "HOST": "http://127.0.0.1",
+    "PORT": 8000,
+    "ROUTES": {
+      "USER_SIGNUP": "/accounts/signup/",
+      "USER_VERIFICATION": "/accounts/verification/",
+      "USER_RESEND_VERIFICATION_CODE": "/accounts/resend-verification-code/",
+      "USER_LOGIN": "/accounts/login/",
+      "USER_LOGOUT": "/accounts/logout/"
```

Comment:
I don't know if we need to routes in configs. Just `HOST` and `PORT` should suffice. I think i'd make more sense for `ROUTES` to be in the ipc files themselves.

Reply:
done